Tests/TemplatesTests/TestsHelper.swift

```diff
@@ -178,7 +178,14 @@ extension XCTestCase {
 
         let result: String
         do {
-          result = try template.render(context)
+          let renderedResult = try template.render(context)
+
+          // Remove `swiftlint:disable all` statement from result
+          // This is needed so that we can still verify the code follows all SwiftLINT guidelines
+          result = renderedResult.replacingOccurrences(
+            of: "// swiftlint:disable all\n\n",
+            with: ""
+          )
```

Comment:
No worries about the review, you really made up for everything with this instant reponse 😉

I tried to implement this approach at first, but I found it more convenient and "clean" to do the modification in the tests helper, as we would otherwise have to manipulate files both prior to and after linting which didn't seem reasonable to me (As far as I know, SwiftLint doesn't allow us to pass texts instead of file locations).

If you believe that's the better solution, I can of course try elaborating.

Sidenote: Maybe it's not even needed to "restore" the files after the modification in the lint script phase, but considering this is a large project, I'd rather be more careful with another generation mechanism kicking in.

Reply:
Ah, fair enough.

But in fact SwiftLint can lint text 😉 there is a `--use-stdin` option in SwiftLint so you can just use `sed` or `grep -v` for example to print the content of a swift file to stdout while replacing/removing the disable comment then pipe that to SwiftLint. That way no file is modified for that. Magic!

So something along the lines of `find /path/of/dir/to/lint -name *.swift -exec (grep -v '// swiftlint:disable all' {} | swiftlint lint --use-stdin`) \;` or something I guess

Reply:
That disables SwiftLint's caching mechanism though, but I'm not sure there's a better solution. Maybe copying the output files to a temp dir, stripping the `disable:all`, and linting that folder? Not sure which is better 🤷‍♂️